docs/ESP_DEVELOPMENT.md

```diff
@@ -0,0 +1,54 @@
+# mLRS Documentation: Compiling and Flashing for ESP82xx Devices #
+
+([back to main page](../README.md))
+
+For ESP devices, the project uses the Arduino framework with Espressif cores.
+
+## Software: Installation Bits and Bops
+
+### Prerequisites ####
+
+1. Install Visual Studio Code, which can be found here: [VSCode](https://code.visualstudio.com/)
+2. Install the PlatformIO extension, which is done within VSCode: [PlatformIO Instructions](https://platformio.org/install/ide?install=vscode)
+3. Install Python 3 and make sure that it is added to your environment/path variable.
+
```

Comment:
git needs also to be installed and in path

Reply:
Added